.travis.yml

```diff
@@ -13,6 +13,7 @@ install:
   - which node
   - clang++ -v
   - which clang++
+  - make distclean
```

Comment:
It appears that travis has started caching by default. This is now needed to ensure that the `mason_packages` is cleared before each build: the `Makefile` looks at whether files exist in `mason_packages` to decide what to install. Clearing it avoids potential partial install problems.

Reply:
👍 for the explicit clear here - another option is to disable caching completely: https://docs.travis-ci.com/user/caching/#explicitly-disabling-caching